pkg/shell/base_index.js

```diff
@@ -940,8 +941,23 @@ function CompiledComponents() {
                     label: cockpit.gettext(info.label) || prop,
                     order: info.order === undefined ? 1000 : info.order,
                     icon: info.icon,
-                    wants: info.wants
+                    wants: info.wants,
+                    keywords: info.keywords || { },
+                    keyword: { score: -1 }
                 };
+
+                // Always first keyword should be page name
+                const general = item.keywords.general || { matches: [] };
+                general.matches.unshift(item.label.toLowerCase());
+                item.keywords.general = general;
+
+                // Also we want keywords to be translated and have higher priority
+                // then content of pages
+                Object.values(item.keywords).map(i => {
```

Comment:
This is hard to read -- you create a new list, but discard it -- what does that code actually do? If that really modifies item.keywords inline, please use `forEach()` instead.

Reply:
this is bogus, needs to be removed and just need to read coef and native from manifest.